scripts/lib/CIME/case.py

```diff
@@ -1108,6 +1108,25 @@ def apply_user_mods(self, user_mods_dir=None):
                 user_mods_path = os.path.join(user_mods_path, user_mods)
             apply_user_mods(self._caseroot, user_mods_path)
 
+    def _get_comp_user_mods(self, component):
+        """
+        For a component 'foo', gets the value of FOO_USER_MODS.
+
+        Returns None if no value was found, or if the value is an empty string.
+        """
+        comp_user_mods = self.get_value("{}_USER_MODS".format(component.upper()))
+        if comp_user_mods is None or comp_user_mods == "":
```

Comment:
Ideally it seems like we'd have some shared handling of empty strings in some common place, but I'm not sure where this should go (e.g., one of the get_value routines somewhere in the inheritance chain?), and I didn't want to risk breaking things this late in the release cycle, so for now I'm just applying this special handling for this one case where I know we need it.